packages/terra-framework-docs/src/terra-dev-site/doc/folder-tree/Examples.2/FolderTreeExpandCollapse.3.doc.mdx

```diff
@@ -0,0 +1,45 @@
+import FolderTreeExpandCollapse from './FolderTreeExpandCollapse?dev-site-example';
+
+# Expand/Collapse
+
+### Description
+
+The following example demonstrates expandable/collapsible folders.
+
+### Properties required
+
+Expanded/collapsed state is controlled by the consuming component.
+The following FolderTreeItem props are required for expand/collapse:
+* **isExpanded**: Identifies whether or not the FolderTreeItem is currently expanded. 
```

Comment:
Same as above, probably should be links to the props table

Reply:
[4b462af](https://github.com/cerner/terra-framework/pull/1973/commits/4b462af48977e5fb036f4ed1a0a9666f2918c9de)